src/nvim/charset.c

```diff
@@ -886,14 +896,13 @@ int vim_isfilec_or_wc(int c)
   return vim_isfilec(c) || c == ']' || path_has_wildcard(buf);
 }
 
-/// return TRUE if 'c' is a printable character
+/// Check that "c" is a printable character.
 /// Assume characters above 0x100 are printable (multi-byte), except for
```

Comment:
This sentence gets a bit confusing as unicode is our default now. maybe "Assume characters above 0x100 are printable for double-byte encodings"